contracts/BroadcastMarket.sol

```diff
@@ -211,12 +284,18 @@ struct RevealRequest {
     address requester;
     address collector;
     uint256 location;
-    uint256 x;
-    uint256 y;
     uint256 payout;
+    uint256 cancelCompleteBlock;
+    bool isKnown;
     bool paid;
     bool refunded;
-    uint256 cancelCompleteBlock;
+}
+
+struct Coord {
+    uint256 x;
+    uint256 y;
+    // wasted space because empty is 0 in solidity
+    bool isOrigin;
```

Comment:
annoying hack because looking up a nonexistant planetid would return a zerod Coords. However 0,0 is a valid coordinate set. So we set isOrigin if anyone ever actually attempts request 0,0.